github/packages/github/github.js

```diff
@@ -12,10 +12,160 @@ const axios = require('axios');
 async function _command(params, commandText, secrets = {}) {
 
   return {
-    response_type: 'in_channel', // or `ephemeral` for private response
-    text: await axios.get('https://raw.githubusercontent.com/nimbella/command-sets/master/github/README.md')
-    .then(response => { return response.data; })
-    .catch(error => { return error.response.data; })
+    "blocks": [
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": "Nimbella Commander GitHub Command Set.\
+          \n_A Nimbella Commander command set for interfacing with GitHub._\
+          \n\n*Features*:\
+          \n• Create an issue\
+          \n• Close an issue\
+          \n• Reopen an issue\
+          \n• Find pull requests by date\
+          \n• Label an issue/pull request\
+          \n• Request someone to review a pull request\
+          \n• View repository community statistics\
+          \n• View recent pull requests\
+          \n• View GitHub command set documentation\
+          \n\n*Install*:\

```

Comment:
We should probably remove this since the help is only available after installation.